build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -155,6 +155,18 @@ public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetConte
 
             if ( ! File.Exists(combinedSharedHostAndFrameworkArchiveDownloadFile))
             {
+                new AzurePublisher().DownloadFile(combinedSharedHostAndFrameworkArchiveFile, 
```

Comment:
This method was static to bypass the environment variable checks in the constructor of AzurePublisher since they're not necessary for htis method to work.